### DISCUSSION TITLE
GitHub Education

### DISCUSSION THREAD

**Discussion by emmanuelmartindavid** [community/community#52936] [GitHub Education] [answered]
Hi. I have a question, maby someone could help me. I started a free-trial last month for copilot. This month, i got aprobe for gitHub education as  student. My question is, will the charge me for copilot when the free-trial ends?
Thank you.

**Comment by samus-aran** [comment, marked as answer]
Hi @emmanuelmartindavid 👋 

No it shouldn't charge you once your free trial ends if you are a verified student 👍 Have fun with Copilot 👩‍✈️

**Reply by emmanuelmartindavid** [reply]
Thanks!!

**Reply by kakar930** [reply]
Hi @emmanuelmartindavid 👋

No it shouldn't charge you once your free trial ends if you are a verified student 👍 Have fun with Copilot

**Reply by kakar930** [reply]
52936

**Reply by kakar930** [reply]
Hi @emmanuelmartindavid 👋

No it shouldn't charge you once your free trial ends if you are a verified student 👍 Have fun with Copilot

**Comment by kakar930** [comment]
Hi @emmanuelmartindavid 👋

No it shouldn't charge you once your free trial ends if you are a verified student 👍 Have fun with Copilot

**Comment by MrdeckA** [comment]
hello
